playbook/app/pb_kits/playbook/pb_fixed_confirmation_toast/fixed_confirmation_toast.rb

```diff
@@ -18,6 +20,10 @@ def close_class
         closeable.present? ? " remove_toast" : ""
       end
 
+      def multi_line_class
+        multi_line.present? ? "_multi_line" : ""
```

Comment:
That extra underscore, I think can be removed if you include the `multi_line_class` inside `generate_classname`. See below.